app/src/main/java/org/sopt/dateroad/data/dataremote/model/response/ResponseAdvertisementsDto.kt

```diff
@@ -0,0 +1,10 @@
+package org.sopt.dateroad.data.dataremote.model.response
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+class ResponseAdvertisementsDto(
+    @SerialName("advertisementDtoResList")
+    val advertisementDtoResList: List<ResponseAdvertisementDto>
```

Comment:
```suggestion
    val advertisements: List<ResponseAdvertisementDto>
```
이런 식으로 네이밍 해주시는 게 좋을 것 같아요! 네이밍에 List 등을 사용하는 건 타입에 대한 의존성을 강조하는 거라서요

Reply:
서버에서 이렇게 내려줘서 List로 설정했어요!

Reply:
서버에서 그렇게 내려줘도 안드 내부에서는 List 제거하고 해당 네이밍을 따라가는 게 좋을 것 같아요 꼭 서버 네이밍을 따라갈 필요는 없으니까요